### ISSUE TITLE
Overwatch gives ability for random player to get you banned over nothing

### ISSUE THREAD

**Issue by JCrick-dotes** [ValveSoftware/Dota2-Gameplay#15985]
### Description

People report in game over the smallest frustrations.  You can be playing perfectly fine but your ally dies and wants you to be the scape goat.  If reviewer of overwatch sees you died a lot that game they can assume you griefed the game and convict.   It's quite ridiculous considering the amount of deaths on a support has a ton to do with the core picks.  Currently I'm banned for 4 hours from playing when I did nothing wrong, played the entire game without giving up.  You need a forum for unjustified bans.  This happens far too often. 

### Screenshots

https://steamcommunity.com/profiles/76561199025322264/gcpd/570?category=Account&tab=MatchPlayerReportIncoming

**Comment by Nallefjant** [comment]
Yup, this happens to me like once a month, I always try my best to win, but you know sometimes you have bad games and you get reported. Developers don't give a fuck about that. It's quite silly that you sometimes have to TP into a losing battle and die with some people just to avoid reports (or getting signed as a griefer by the reviewer).

**Comment by TobyFlendersonFromHR** [comment]
maybe trying your best is still the equivalent of playing trash

sorry.
